README.md

```diff
@@ -48,6 +48,7 @@ The old NuGet package has been unlisted and will not receive any updates any mor
     - [setHttpHeader](#sethttpheader)
     - [setBody](#setbody)
     - [setBodyAsString](#setbodyasstring)
+    - [redirectTo](#redirectTo)
```

Comment:
Maybe you can add the documentation of `redirectdTo` more towards the end after `negotiate` then it matches the order you also added in the HttpHandlers.fs file. Thank you!